Src/WitsmlExplorer.Frontend/components/Sidebar/WellboreItem.tsx

```diff
@@ -117,7 +117,7 @@ const WellboreItem = (props: WellboreItemProps): React.ReactElement => {
           </ObjectGroupItem>
           <ObjectGroupItem objectType={ObjectType.Message} />
           <ObjectGroupItem objectsOnWellbore={wellbore?.mudLogs} objectType={ObjectType.MudLog} ObjectContextMenu={MudLogContextMenu} />
-          <ObjectGroupItem objectType={ObjectType.Rig} />
+          <ObjectGroupItem objectsOnWellbore={wellbore?.rigs} objectType={ObjectType.Rig} />
```

Comment:
Right clicking a rig crashes the app. `ObjectContextMenu={RigContextMenu}` is missing on this line, as specified on line 24 in ObjectGroupItem.

Reply:
Ok, thank you for finding. Fixed.